packages/E2ETest/wdio.conf.js

```diff
@@ -239,12 +248,8 @@ exports.config = {
    * Function to be executed after a test (in Mocha/Jasmine) or a step (in Cucumber) starts.
    * @param {Object} test test details
    */
-  afterTest: function(test) {
-    if (test.error !== undefined) {
-      let name = 'ERROR-' + Date.now();
-      browser.saveScreenshot('./errorShots/' + name + '.png');
-    }
-  },
+  // afterTest: function(test) {
+  // },
```

Comment:
remove?

Reply:
Will keep it. it's from the template, and the template have beforeTest and other functions

---
In reply to: [539642640](https://github.com/microsoft/react-native-windows/pull/6636#discussion_r539642640) [](ancestors = 539642640)